### ISSUE TITLE
Unable to see any outputs in Jupyter Notebook

### ISSUE THREAD

**Issue by spamula** [uber/manifold#98]
I've installed the mlvis extension and followed the steps installing the nbextension and then enabling it. 

When following the manifold example notebook (https://github.com/uber/manifold/blob/master/bindings/jupyter/notebooks/manifold.ipynb) and running the first cell I get the output:

Manifold(props='{"data": {"x": [{"feature_0": 318, "feature_1": "A"}, {"feature_0": 213, "feature_1": "B"}, {"…

And no visual output. Is there a step that I need to change to get the notebook visual output working?

I've even tried disabling the other extensions to see whether it would work and no change. 

I'm running jupyter version 1.0.

**Comment by kenns29** [comment]
@spamula I cannot reproduce this error. Do you mind do a `pip freeze` to print out all the dependencies and check your browser console to see if there are any error messages?

**Comment by kenns29** [comment]
I noticed that the Jupyter wrapper may have issues with the `conda` environment. I will look into this issue.

**Comment by spamula** [comment]
Hi @kenns29 

Here's the pip freeze:

absl-py==0.5.0
addfips==0.2.2
affine==2.3.0
alabaster==0.7.12
alembic==1.0.5
amqp==2.3.2
anyconfig==0.9.7
apache-beam==2.15.0
apache-libcloud==2.5.0
aplus==0.11.0
arcgis==1.6.0
argh==0.26.2
arrow==0.12.1
asn1crypto==0.24.0
astor==0.7.1
astroid==2.2.5
astropy==3.1.2
atomicwrites==1.3.0
attrs==19.1.0
autopep8==1.3.5
avro-python3==1.9.1
azure-common==1.1.18
azure-storage-blob==1.5.0
azure-storage-common==1.4.0
Babel==2.6.0
backcall==0.1.0
backports.os==0.1.1
backports.shutil-get-terminal-size==1.0.0
beautifulsoup4==4.7.1
billiard==3.5.0.5
binaryornot==0.4.4
bitarray==0.8.3
bkcharts==0.2
blaze==0.10.1
bleach==3.1.0
bokeh==1.0.4
boto==2.49.0
boto3==1.9.103
botocore==1.12.103
Bottleneck==1.2.1
bqplot==0.11.6
branca==0.3.1
bz2file==0.98
CacheControl==0.12.5
cachetools==3.1.1
cchardet==2.1.4
celery==4.2.1
certifi==2019.3.9
cffi==1.12.2
chardet==3.0.4
cleanco==1.3
Click==7.0
click-plugins==1.0.4
cligj==0.5.0
cloudpickle==0.8.0
colorama==0.4.1
coloredlogs==10.0
colorlover==0.2.1
contextily==1.0rc2
contextlib2==0.5.5
convertdate==2.1.3
cookiecutter==1.6.0
crcmod==1.7
cryptography==2.6.1
cufflinks==0.14.6
cycler==0.10.0
cymem==1.31.2
Cython==0.29.11
cytoolz==0.9.0.1
dash==1.7.0
dash-core-components==1.6.0
dash-cytoscape==0.1.1
dash-daq==0.1.4
dash-html-components==1.0.2
dash-renderer==1.2.2
dash-table==4.5.1
dash-table-experiments==0.6.0
dask==1.2.2
datashape==0.5.2
dateparser==0.7.1
decorator==4.4.0
decore==0.0.1
defusedxml==0.6.0
Deprecated==1.2.5
descartes==1.1.0
dill==0.2.9
Distance==0.1.3
distributed==1.26.0
docopt==0.6.2
docutils==0.14
elasticsearch==6.3.1
eli5==0.8
entrypoints==0.3
enum34==1.1.6
ephem==3.7.6.0
et-xmlfile==1.0.1
ez-setup==0.9
fastavro==0.21.24
fastcache==1.0.2
fasteners==0.15
featuretools==0.13.0
filelock==2.0.13
Fiona==1.8.0
Flask==1.0.3
Flask-AppBuilder==1.12.2
Flask-Babel==0.12.2
Flask-Cache==0.13.1
Flask-Caching==1.4.0
Flask-Compress==1.4.0
Flask-Cors==3.0.8
Flask-Login==0.4.1
Flask-Migrate==2.3.1
Flask-OpenID==1.2.5
Flask-Script==2.0.6
Flask-SQLAlchemy==2.3.2
Flask-Testing==0.7.1
Flask-WTF==0.14.2
flower==0.9.2
fsspec==0.6.1
funcsigs==1.0.2
furl==2.0.0
future==0.16.0
fuzzywuzzy==0.16.0
gast==0.2.2
gcsfs==0.2.3
gekko==0.1rc3
gender-guesser==0.4.0
gensim==3.5.0
geographiclib==1.49
geojson==2.4.1
geopandas==0.5.1
geopy==1.18.1
gevent==1.4.0
gitdb2==2.0.5
GitPython==2.1.11
glob2==0.6
google-api-core==1.12.0
google-api-python-client==1.7.11
google-apitools==0.5.28
google-auth==1.6.3
google-auth-httplib2==0.0.3
google-auth-oauthlib==0.4.1
google-cloud-bigquery==1.16.0
google-cloud-bigtable==0.32.2
google-cloud-core==1.0.2
google-cloud-datastore==1.7.4
google-cloud-pubsub==0.39.1
google-cloud-storage==1.16.1
google-pasta==0.1.8
google-resumable-media==0.3.2
googleapis-common-protos==1.6.0
googlemaps==3.1.4
graphviz==0.13.2
greenlet==0.4.15
grpc-google-iam-v1==0.11.4
grpcio==1.15.0
gunicorn==19.9.0
h5py==2.9.0
hdbscan==0.8.15
hdfs==2.5.8
HeapDict==1.0.0
holidays==0.9.10
html5lib==1.0.1
httplib2==0.12.0
humanfriendly==4.18
humanize==0.5.1
hyperopt==0.1.2
icc-rt==2019.0
idna==2.8
ijson==2.3
imageio==2.5.0
imagesize==1.1.0
iml==0.6.0
importlib-metadata==0.0
intel-openmp==2019.0
ipydatawidgets==4.0.1
ipykernel==5.1.1
ipyleaflet==0.11.1
ipympl==0.3.3
ipython==7.5.0
ipython-genutils==0.2.0
ipyvolume==0.5.2
ipyvuetify==0.1.7
ipywebrtc==0.5.0
ipywidgets==7.5.1
iso3166==0.8
isodate==0.6.0
isort==4.3.16
itsdangerous==1.1.0
jdcal==1.4
jedi==0.13.3
jeepney==0.4
jellyfish==0.7.1
Jinja2==2.10.1
jinja2-time==0.2.0
jmespath==0.9.3
joblib==0.13.2
jsonlines==1.2.0
jsonmodels==2.4
jsonpickle==0.9.6
jsonschema==3.0.1
jupyter==1.0.0
jupyter-client==5.2.4
jupyter-console==6.0.0
jupyter-contrib-core==0.3.3
jupyter-contrib-nbextensions==0.5.0
jupyter-core==4.4.0
jupyter-highlight-selected-word==0.2.0
jupyter-latex-envs==1.4.4
jupyter-nbextensions-configurator==0.4.1
jupyterlab==0.35.4
jupyterlab-launcher==0.10.2
jupyterlab-server==0.2.0
jupyterthemes==0.20.0
kedro==0.14.1
keplergl==0.1.0a8
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
keyring==19.0.1
kiwisolver==1.1.0
kmodes==0.9
kombu==4.2.2.post1
landsatxplore==0.0.5
lazy-object-proxy==1.3.1
lesscpy==0.13.0
libpagure==0.9
lightgbm==2.3.1
lime==0.1.1.34
linear-tsv==1.1.0
llvmlite==0.28.0
locket==0.2.0
lunardate==0.2.0
lxml==4.3.2
Mako==1.0.7
mapbox==0.18.0
Markdown==3.0.1
MarkupSafe==1.1.1
matplotlib==3.1.0
matplotlib-venn==0.11.5
mccabe==0.6.1
memory-profiler==0.54.0
mercantile==1.1.2
missingno==0.4.1
mistune==0.8.4
mkl==2019.0
mkl-fft==1.0.6
mkl-random==1.0.1.1
mlvis==0.0.9
mock==2.0.0
monotonic==1.5
more-itertools==7.0.0
mpmath==1.1.0
msgpack==0.6.1
multipledispatch==0.6.0
munch==2.3.2
murmurhash==0.28.0
mysql-connector==2.1.7
nameparser==0.5.7
nbconvert==5.5.0
nbformat==4.4.0
networkx==2.3
nltk==3.4
nose==1.3.7
notebook==5.7.8
ntlm-auth==1.2.0
numba==0.43.1
numexpr==2.6.9
numpy==1.17.4
numpydoc==0.8.0
oauth2client==4.1.3
oauthlib==3.0.1
odo==0.5.0
ogr==0.0.3
olefile==0.46
opt-einsum==3.1.0
orderedmultidict==1.0
packaging==19.0
pandas==0.25.3
pandas-gbq==0.10.0
pandocfilters==1.4.2
parso==0.4.0
partd==1.0.0
patsy==0.5.1
pbr==5.4.3
pdpipe==0.0.35
pexpect==4.7.0
pickleshare==0.7.5
Pillow==6.0.0
plotly==4.4.1
plotly-express==0.4.1
plotly-geo==1.0.0
pluggy==0.11.0
ply==3.11
polyline==1.3.2
poyo==0.4.1
probableparsing==0.0.1
progressbar2==3.42.0
prometheus-client==0.6.0
prompt-toolkit==2.0.9
protobuf==3.6.1
psutil==5.6.3
ptyprocess==0.6.0
py==1.8.0
pyarrow==0.12.0
pyasn1==0.4.5
pyasn1-modules==0.2.5
pycodestyle==2.5.0
pycparser==2.19
pydata-google-auth==0.1.3
pydot==1.4.1
pydotplus==2.0.2
PyGithub==1.43.5
Pygments==2.4.0
PyJWT==1.7.1
pymongo==3.9.0
PyOpenGL==3.1.0
pyparsing==2.4.0
pyproj==1.9.5.1
pyrsistent==0.15.2
pyshp==2.1.0
pystan==2.18.1.0
pytest==4.5.0
pytest-runner==4.4
python-crfsuite==0.9.6
python-dateutil==2.8.0
python-editor==1.0.3
python-gitlab==1.8.0
python-json-logger==0.1.9
python-utils==2.3.0
python3-openid==3.1.0
pythreejs==2.1.1
pytz==2019.1
PyWavelets==1.0.3
PyYAML==3.13
pyzmq==18.0.1
qtconsole==4.4.3
rasterio==1.1.1
recordlinkage==0.13.2
regex==2019.3.12
requests==2.22.0
requests-oauthlib==1.2.0
retrying==1.3.3
rsa==4.0
Rtree==0.8.3
s3fs==0.4.0
s3transfer==0.2.0
SALib==1.3.4
scikit-image==0.15.0
scikit-learn==0.21.1
scikit-plot==0.3.7
scipy==1.3.0
seaborn==0.9.0
SecretStorage==3.1.1
Send2Trash==1.5.0
setuptools-git==1.2
shap==0.29.1
Shapely==1.6.4.post2
singledispatch==3.4.0.3
six==1.12.0
sklearn==0.0
skope-rules==1.0.0
skutil==0.0.16
smart-open==1.9.0
smmap2==2.0.5
snowballstemmer==1.2.1
snuggs==1.4.7
sortedcontainers==2.1.0
soupsieve==1.8
Sphinx==1.8.5
sphinxcontrib-websupport==1.1.0
SQLAlchemy==1.3.5
statsmodels==0.10.2
strct==0.0.30
swifter==0.289
tables==3.5.1
tabulate==0.8.3
tbb==2019.0
tbb4py==2019.0
tblib==1.3.2
tensorboard==2.0.1
tensorflow==2.0.0
tensorflow-estimator==2.0.1
tensorflow-serving-api==2.0.0
termcolor==1.1.0
terminado==0.8.1
testpath==0.4.2
toolz==0.9.0
toposort==1.5
tornado==6.0.2
tqdm==4.32.1
traitlets==4.3.2
traittypes==0.2.1
typed-ast==1.3.1
typing==3.6.6
tzlocal==1.5.1
uritemplate==3.0.0
urllib3==1.25.2
usaddress==0.5.10
vaex==2.0.2
vaex-arrow==0.3.5
vaex-astro==0.5.0
vaex-core==0.9.2
vaex-hdf5==0.5.2
vaex-jupyter==0.3.0
vaex-ml==0.5.0
vaex-server==0.2.1
vaex-ui==0.2.2
vaex-viz==0.3.4
vine==1.1.4
watermark==1.8.1
wcwidth==0.1.7
webencodings==0.5.1
Werkzeug==0.15.4
whichcraft==0.4.1
widgetsnbextension==3.5.0
witwidget==1.5.0
wrapt==1.11.1
WTForms==2.2.1
xarray==0.12.3
xgboost==0.90
xlrd==1.2.0
XlsxWriter==1.1.5
yellowbrick==0.9.1
zict==0.1.4

The jupyter nbextension list is as below:

Known nbextensions:
  config dir: /home/XXX.jupyter/nbconfig
    notebook section
      nbextensions_configurator/config_menu/main  enabled 
      - Validating: problems found:
        - require?  X nbextensions_configurator/config_menu/main
      contrib_nbextensions_help_item/main disabled
      execute_time/ExecuteTime  enabled 
      - Validating: OK
      hide_input/main disabled
      toc2/main  enabled 
      - Validating: OK
      code_prettify/autopep8 disabled
      scratchpad/main disabled
      snippets/main disabled
      jupyter-webrtc/extension disabled
      jupyter-matplotlib/extension disabled
      jupyter-leaflet/extension disabled
      ipyvolume/extension disabled
      jupyter-vuetify/extension disabled
      jupyter-js-widgets/extension disabled
      jupyter-datawidgets/extension disabled
      bqplot/extension disabled
      arcgis/extension disabled
      jupyter-threejs/extension disabled
      plotlywidget/extension disabled
      zenmode/main disabled
      table_beautifier/main disabled
      keplergl-jupyter/extension disabled
      wit-widget/extension disabled
      mlvis/extension  enabled 
      - Validating: OK
    tree section
      nbextensions_configurator/tree_tab/main  enabled 
      - Validating: problems found:
        - require?  X nbextensions_configurator/tree_tab/main
  config dir: /home/XXX/floodenv/etc/jupyter/nbconfig
    notebook section
      bqplot/extension  enabled 
      - Validating: OK
      ipyvolume/extension  enabled 
      - Validating: OK
      jupyter-webrtc/extension  enabled 
      - Validating: OK
      jupyter-datawidgets/extension  enabled 
      - Validating: OK
      jupyter-leaflet/extension  enabled 
      - Validating: OK
      jupyter-matplotlib/extension  enabled 
      - Validating: OK
      jupyter-threejs/extension  enabled 
      - Validating: OK
      jupyter-vuetify/extension  enabled 
      - Validating: OK
      keplergl-jupyter/extension  enabled 
      - Validating: OK
      plotlywidget/extension  enabled 
      - Validating: OK
      jupyter-js-widgets/extension  enabled 
      - Validating: OK
      wit-widget/extension  enabled 
      - Validating: OK
      arcgis/extension  enabled 
      - Validating: OK
      mlvis/extension  enabled 
      - Validating: OK

**Comment by kenns29** [comment]
@spamula try to enable the `jupyter-js-widgets` nbextension. It is a dependency.

**Comment by spamula** [comment]
I'm not sure how I missed this with the install but got the jupyter-js-widgets set up and all working now @kenns29 thanks!

**Comment by Casyfill** [comment]
@spamula how did you do that?

**Comment by Firenze11** [comment]
Closing the issue since it is resolved.
